packages/ui/src/components/View/SourceTargetView.tsx

```diff
@@ -24,7 +24,11 @@ export const SourceTargetView: FunctionComponent = () => {
   const { targetBodyDocument } = useDataMapper();
   const { reloadNodeReferences, setDefaultHandler, setMappingLinkCanvasRef } = useCanvas();
   const mappingLinkCanvasRef = useRef<HTMLDivElement>(null);
-  setMappingLinkCanvasRef(mappingLinkCanvasRef);
+
+  useEffect(() => {
+    setMappingLinkCanvasRef(mappingLinkCanvasRef);
```

Comment:
is this something that has to be set only once ? (after first render)

Reply:
yes